sklearn/linear_model/logistic.py

```diff
@@ -1788,69 +1873,88 @@ def fit(self, X, y, sample_weight=None):
                       intercept_scaling=self.intercept_scaling,
                       random_state=self.random_state,
                       max_squared_sum=max_squared_sum,
-                      sample_weight=sample_weight
+                      sample_weight=sample_weight,
+                      l1_ratio=l1_ratio
                       )
             for label in iter_encoded_labels
-            for train, test in folds)
-
+            for train, test in folds
+            for l1_ratio in l1_ratios_)
+
+        # _log_reg_scoring_path will output different shapes depending on the
+        # multi_class param, so we need to reshape the outputs accordingly.
+        # After reshaping,
+        # - scores is of shape (n_classes X n_folds X n_Cs . n_l1_ratios)
```

Comment:
What's the shape of Cs and why do we always have to take C[0]?

Reply:
Cs is of shape `(n_classes . n_folds . n_l1_ratios, n_Cs)` and all the rows are equal, so we just take the first one.

I added a comment.